ocviapy/__init__.py

```diff
@@ -543,6 +543,16 @@ def details_str(self):
             )
         return detail_msg
 
+    @property
+    def image_pull_error(self):
+        status = self.data.get("status", {})
+        if status.get("containerStatuses"):
+            for container in status.get("containerStatuses"):
+                reason = container.get("state", {}).get("waiting", {}).get("reason", "")
+                if reason == "ImagePullBackOff" or reason == "ErrImagePull":
```

Comment:
Apparently there's one more possible state (which we probably rarely hit):

```suggestion
                if reason in ("ImagePullBackOff", "ErrImagePull", "ErrImageNeverPull"):
```

Reply:
Added in last push